web_external/js/views/body/AnalysisPanel.js

```diff
@@ -10,7 +10,7 @@
         return name  + 'Widget';
     };
 
-    minerva.views.AnalysisPanel = minerva.View.extend({
+    minerva.views.AnalysisPanel = minerva.views.Panel.extend({
```

Comment:
You are right, and I think what you've done already is cleanest.  Can you add the line 

    _.extend(this.events, minerva.views.Panel.prototype.events);

to the Extending Minerva docs for great explicitness?